app/models/experiment.rb

```diff
@@ -0,0 +1,8 @@
+class Experiment < ApplicationRecord
+  has_many :scientist_experiments
+  has_many :scientists, through: :scientist_experiments
+
+  def long_running_experiments
+    Experiment.where("num_months > 6")
+  end
```

Comment:
Not sure if you were coming back to this method, but the names of the experiments should be in desc order.